proxy/coco/src/peer.rs

```diff
@@ -113,6 +113,20 @@ impl Api {
         Ok(())
     }
 
+    /// Check the storage to see if we have the given commit for project at `urn`.
+    ///
+    /// # Errors
+    ///
+    ///   * Checking the storage for the commit fails.
+    ///
+    /// # Panics
+    ///
+    ///   * Unable to acquire the lock.
+    pub fn has_commit(&self, urn: &RadUrn, oid: impl Into<git2::Oid>) -> Result<bool, Error> {
```

Comment:
We have `coco::oid` now and probably should use that in the public api of this crate. It supports `into` to `git2::Oid`.

Reply:
Aw yis, saw that change but forgot. Thanks :+1:

Reply:
It's kind of weird that I would take `coco::Oid` just call `into` and then in my test case do `coco::Oid::from`, it's just a no-op. So I think leaving this as `Into<git2::Oid>` is fine, because then it can take a `coco::Oid` _or_ a `git2::Oid`.

Reply:
DIdn't see the `Into` part. Alles gut.